analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.CSharp11.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Tests.Diagnostics
```

Comment:
Please extract this into separate PR as described on the issue itself. There's too much to review